partials/shop-product-loop.htm

```diff
@@ -1,16 +1,17 @@
 {% set shop_item = theme.shopItemsPerPage %}
```

Comment:
Can probably remove this first set of `shop_item`, it's getting overwritten in the next line anyways.